libs/expo/betterangels/src/lib/ui-components/NoteCard/NoteCardClient.tsx

```diff
@@ -35,6 +35,11 @@ export default function NoteCardClient(props: INoteCardClientProps) {
           accessibilityHint={
             `${displayDetails?.email} client's avatar` || `client's avatar`
           }
+          imageUrl={
+            displayDetails?.__typename === 'ClientProfileType'
+              ? displayDetails.profilePhoto?.url
+              : ''
+          }
```

Comment:
this can be simplified to
```suggestion
          imageUrl={clientProfile ? clientProfile.profilePhoto?.url : ''}
```